src/functions.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Lunanimous\Rpc;
+
+if (!function_exists('toNim')) {
+    /**
+     * Converts from luna to nim.
+     *
+     * @param int $luna the value in luna (smallest unit)
+     *
+     * @return float the value in nim
+     */
+    function toNim($luna)
+    {
+        return $luna / 100000;
+    }
```

Comment:
Maybe it's valuable to make sure the input is an `int`, so that we can be sure the result has a maximum of 5 decimal places.